exo-go/features/run/app-dependency.feature

```diff
@@ -0,0 +1,23 @@
+Feature: running Exosphere applications with dependencies
+
+  As an Exosphere developer
+  I want to have an easy way to run application wide dependencies
+  So that all my services can use these dependencies.
+
+  Rules:
+  - run "exo run" in the directory of your application to run it
```

Comment:
This is not a rule for the "application-wide dependencies" feature. The rules are:

- list application dependencies in the "dependencies" key of "application.yml"
- these dependencies get booted up before the application services